.circleci/config.yml

```diff
@@ -0,0 +1,100 @@
+version: 2.1
+orbs:
+  node: circleci/node@5.0.2
+jobs:
+  build:
+    executor: node/default
+    steps:
+      - checkout
+      - node/install-packages:
+          pkg-manager: npm
+      - run:
+          command: npm install
+          name: Install npm
+      - run:
+          command: sudo npm install -g ts-node
+          name: Install ts-node
+      - run:
+          command: ts-node -v
+          name: Check ts-node version to make sure it installed
+  test:
+    executor: node/default
+    steps:
+      - checkout
+      - node/install-packages:
+          pkg-manager: npm
+      - run:
```

Comment:
You shouldn't do this at every stage –– `npm install` is costly time-wise. You should instead persist this data to the workspace: see `persist_to_workspace` in our other repos